cfsdk/src/main/java/io/harness/cfsdk/cloud/Cloud.java

```diff
@@ -139,8 +139,11 @@ private void authenticate() throws ApiException {
         this.authInfo = authResponseDecoder.extractInfo(authToken);
 
         if (authInfo != null) {
-            apiClient.addDefaultHeader("Harness-EnvironmentID", authInfo.getEnvironmentIdentifier());
-            apiClient.addDefaultHeader("Harness-AccountID", authInfo.getAccountID());
+            apiClient.addDefaultHeader("Harness-EnvironmentID", authInfo.getEnvironment());
```

Comment:
Same as above. We don't want the `dev` , `prod` etc. identifier here, but the UUID